api_tests/sanity/basic_lnd_payment.ts

```diff
@@ -1,24 +1,30 @@
 import { twoActorTest } from "../lib_sdk/actor_test";
+import { LedgerKind } from "../lib_sdk/ledger";
+import { AssetKind } from "../lib_sdk/asset";
+import { expect } from "chai";
 
 setTimeout(function() {
     twoActorTest("sanity-lnd-alice-pays-bob", async function({ alice, bob }) {
-        //
-        // Do a bunch of initialisation like we do in `sendRequest` e.g., init/fund wallet
-        //
-
-        await alice.startLnd();
-        await bob.startLnd();
+        await alice.sendRequest(
+            { ledger: LedgerKind.Lightning, asset: AssetKind.Bitcoin },
+            { ledger: LedgerKind.Bitcoin, asset: AssetKind.Bitcoin }
+        );
 
-        await alice.lnd.fund(); // Send to address: bitcoin wallet -> LND wallet
+        const alicePeers = await alice.wallets
```

Comment:
Reaching this deep into objects makes the tests hard to read and brittle to refactoring of the internal structure.

I'd suggest to provide an accessor for the LN peers on `Actor`.

Reply:
These are only temporary assertions to show that the PR does what it says it does.

Such assertions should be remove at the next PR that does the `addInvoice` and `sendPayments`.